test/unit/resources/aws_iam_root_user_test.rb

```diff
@@ -44,4 +44,52 @@ def test_has_mfa_enabled_returns_false_when_account_mfa_devices_is_zero
 
     assert_equal false, AwsIamRootUser.new(@mock_conn).has_mfa_enabled?
   end
-end
+
+  def test_has_virtual_mfa_enabled_returns_true_when_account_vmfa_devices_is_one
+    test_list_virtual_mfa_devices = OpenStruct.new(
+      virtual_mfa_devices: [Aws::IAM::Types::VirtualMFADevice.new(
+        serial_number: "arn:aws:iam::123456789011:mfa/root-account-mfa-device",
+        user: Aws::IAM::Types::User.new(
+          user_id: "123456789011",
+          arn:     "arn:aws:iam::123456789011:root",
+        )
+      )]
+    )
+    @mock_client.expect :list_virtual_mfa_devices, test_list_virtual_mfa_devices
+
+    assert_equal true, AwsIamRootUser.new(@mock_conn).has_virtual_mfa_devices?
+  end
+
+  def test_has_virtual_mfa_enabled_returns_false_when_account_vmfa_devices_is_zero
+    test_list_virtual_mfa_devices = OpenStruct.new(
+      virtual_mfa_devices: []
+    )
+    @mock_client.expect :list_virtual_mfa_devices, test_list_virtual_mfa_devices
+
+    assert_equal false, AwsIamRootUser.new(@mock_conn).has_virtual_mfa_devices?
+  end
+
+  def test_has_hardware_mfa_enabled_returns_true_when_account_hardware_devices_is_one
+    test_list_hardware_mfa_devices = OpenStruct.new(
+      mfa_devices: [Aws::IAM::Types::MFADevice.new(
+        serial_number: "arn:aws:iam::123456789000:mfa/root-account-mfa-device",
+        user_name: Aws::IAM::Types::User.new(
+          user_id: "123456789000",
+          arn:     "arn:aws:iam::123456789000:root",
+        )
+      )]
+    )
+    @mock_client.expect :list_mfa_devices, test_list_hardware_mfa_devices
+
+    assert_equal true, AwsIamRootUser.new(@mock_conn).has_hardware_mfa_devices?
+  end
+
+  def test_has_hardware_mfa_enabled_returns_false_when_account_hardware_devices_is_zero
+    test_list_hardware_mfa_devices = OpenStruct.new(
+      mfa_devices: []
+    )
+    @mock_client.expect :list_mfa_devices, test_list_hardware_mfa_devices
+
+    assert_equal false, AwsIamRootUser.new(@mock_conn).has_hardware_mfa_devices?
+  end
+end
```

Comment:
One missing piece: you need to add the two new matchers to the docs, at `docs/resources/aws_iam_root_account.md.erb`.  Thanks!